compiler/src/test.rs

```diff
@@ -335,12 +336,35 @@ impl Namespace for CompileNamespace {
     }
 }
 
+impl Namespace for ImportNamespace {
+    fn parse_type(&self) -> ParseType {
+        ParseType::Whole
+    }
+
+    fn run_test(&self, test: Test) -> Result<Value, String> {
+        let err_buf = BufferEmitter(Rc::default());
+        let handler = Handler::new(Box::new(err_buf.clone()));
+
+        // In import tests we only keep Error code to make error messages uniform accross
+        // all platforms and exclude all platform-specific paths.
```

Comment:
Wouldn't it be preferable to sanitize paths so that we can actually see error outputs? (what rustc's compiletest does)

Reply:
Well, we have multiple problems here. 

1. Sanitization way doesn't work well with Windows, as with Windows we sometimes get paths like: ```File: D:\a\leo\leo\test-framework\../tests/compiler\import_misc\names_dash_a_fail.leo``` I've tried sanitizing those, and the number of cases we need to cover is way to extreme. 
2. Windows' OS errors result in different error codes than on Ubuntu/Mac. So we need to additionally wrap up OS errors and unify them too.

To summarize (we've also briefly discussed it with @acoglio) having a simplified expectation in this case is the most painful way. For making sure that the error we get here is the one definitely enough.

Reply:
So compiletest also needs to deal with Windows, so you might want to look at how it handles sanitizing.

Reply:
@Centril you mean Compile namespace? We don't use real paths in the Compile namespace; that is the difference between Import and Compile namespace, the former needs CWD setting while the latter is okay as it is.

Reply:
Oh. Got it. You're talking about rustc.